llms/mlx_lm/cache_prompt.py

```diff
@@ -165,14 +166,13 @@ def main():
         )
 
     print()
-    print(f"Peak memory: {mx.metal.get_peak_memory() / 2**30:.3f} GB")
+    print(f"Peak memory: {mx.metal.get_peak_memory() / 1e9:.3f} GB")
```

Comment:
Changed all of these to `1e9` as technically that's GB and we were slightly under counting.